Windows/SettingsWindow.xaml

```diff
@@ -20,192 +20,257 @@
-                        <Style TargetType="Border" >
-                            <Setter Property="Padding" Value="5,5,5,5" />
-                        </Style>
-                    </Grid.Resources>
-                    <Grid.RowDefinitions>
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                        <RowDefinition Height="*" />
-                    </Grid.RowDefinitions>
-                    <Grid.ColumnDefinitions>
-                        <ColumnDefinition Width="110" />
-                        <ColumnDefinition Width="110" />
-                    </Grid.ColumnDefinitions>
-
-                    <Border Grid.Row="0" Grid.Column="0">
-                        <TextBlock VerticalAlignment="Center">Show in Discord</TextBlock>
-                    </Border>
-
-                    <Border Grid.Row="0" Grid.Column="1">
-                        <CheckBox x:Name="checkDiscord" Click="CheckDiscord_Click"/>
-                    </Border>
-
-                    <Border Grid.Row="1" Grid.Column="0">
-                        <TextBlock VerticalAlignment="Center">Check for Updates</TextBlock>
-                    </Border>
-
-                    <Border Grid.Row="1" Grid.Column="1">
-                        <CheckBox x:Name="checkStartupUpdate" Click="CheckStartupUpdate_Click"/>
-                    </Border>
-
-                    <Border Grid.Row="2" Grid.Column="0">
-                        <TextBlock VerticalAlignment="Center">Map Save Location</TextBlock>
-                    </Border>
-
-                    <Border Grid.Row="2" Grid.Column="1">
-                        <ComboBox x:Name="comboMapSaveFolder" VerticalAlignment="Center" SelectionChanged="comboMapSaveFolder_SelectionChanged">
-                            <ComboBoxItem x:Name="comboMapSaveFolder_Documents">Documents</ComboBoxItem>
-                            <ComboBoxItem x:Name="comboMapSaveFolder_GameInstall">Game Install</ComboBoxItem>
-                        </ComboBox>
-                    </Border>
-
-                    <Border Grid.Row="3" Grid.ColumnSpan="2" Padding="0">
-                        <TextBlock Width="210" FontSize="10" FontStyle="Italic" x:Name="txtMapSaveFolderPath" VerticalAlignment="Center" MouseLeftButtonUp="txtMapSaveFolderPath_MouseLeftButtonUp" Foreground="{DynamicResource {x:Static SystemColors.HotTrackBrushKey}}">C:/Username/Documents/Ragnarock/CustomSongs</TextBlock>
-                    </Border>
-                </Grid>
-            </StackPanel>
+            <ScrollViewer VerticalScrollBarVisibility="Auto" FocusVisualStyle="{x:Null}">
```

Comment:
I've added scrolling for the settings, as the new section was quite long and it didn't make sense to make the window even longer.